docs/spec/spec.md

```diff
@@ -180,3 +181,38 @@ Behaviour of the `push` command,
 - User should add the relevant configuration to the Ballerina.toml file.
 - The user should have initiated the persistence layer in the project and executed the `generate` command to generate the SQL script.
 - If the user invokes the command twice, it will not fail. It will rerun the SQL script against the database.
+
+## 5. Pull Persistence Schema from the Data Provider
+
+```bash
+bal persist pull --datastore mysql --host localhost --port 3306 --user root --database persist
+```
+| Command Parameter |                                      Description                                      | Mandatory | Default Value |
+|:-----------------:|:-------------------------------------------------------------------------------------:|:---------:|:-------------:|
+|    --datastore    | used to indicate the preferred database client. Currently, only 'mysql' is supported. |    No     |     mysql     |
+|      --host       |                          used to indicate the database host                           |    Yes    |     None      |
+|      --port       |                          used to indicate the database port                           |    No     |     3306      |
+|      --user       |                          used to indicate the database user                           |    Yes    |     None      |
+|    --database     |                          used to indicate the database name                           |    Yes    |     None      |
+
+This command will introspect the schema of the database and create a `model.bal` file with the entities and relations based on the schema of the database. The database configuration should be provided as command-line arguments.
+
+
+The file structure of the project should be similar to the following before running the command.
+```
+medical-center
+├── Ballerina.toml
+└── main.bal
+```
```

Comment:
```suggestion
This command should execute within a Ballerina project.
```

Reply:
file structure may vary. we can just say the command should execute with a Ballerina project.